Bottom: 8044ea0aadcd6087aeaa2e84d46a16960f504fb6
Top:    8b316485f1b2a2a46aa139e848a4bf070ca2d8c9
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-06-08 08:52:42 -0500

Refresh of playbook-extract-docker-hosts

---
diff --git a/ansible-playbook/site.yml b/ansible-playbook/site.yml
index b7bee5ae..7cd187c5 100644
--- a/ansible-playbook/site.yml
+++ b/ansible-playbook/site.yml
@@ -18,7 +18,8 @@
   - role: bacula-script
     assemble_scripts: false
   - role: bacula-catalog
-- name: Install/Setup Docker images and containers
+
+- name: Prepare Docker host systems
   hosts: docker-hosts
   remote_user: ansibleremote
   become: true
@@ -33,6 +34,14 @@
     role: docker-common
     tags:
     - docker_common
+
+- name: Install/Setup Docker images and containers on daedalus
+  hosts: daedalus
+  remote_user: ansibleremote
+  become: true
+  become_user: root
+  become_method: sudo
+  roles:
   - name: Configure Docker networking
     role: docker-networking
     net_connection: '{{ daedalus_net_connection }}'
@@ -90,6 +99,10 @@
 
 - name: Setup the Build System (Test Station) with repo data
   hosts: buildsystem
+  remote_user: ansibleremote
+  become: true
+  become_user: root
+  become_method: sudo
   roles:
   - role: buildsystem
     tags:
